rust/lance/benches/take.rs

```diff
@@ -170,10 +308,10 @@ criterion_group!(
         .sample_size(10000)
         .warm_up_time(Duration::from_secs_f32(3.0))
         .with_profiler(PProfProfiler::new(100, Output::Flamegraph(None)));
-    targets = bench_random_take);
+    targets = bench_random_take_with_dataset, bench_random_take_with_file_fragment, bench_random_take_with_file_reader);
 #[cfg(not(target_os = "linux"))]
 criterion_group!(
     name=benches;
     config = Criterion::default().significance_level(0.1).sample_size(10);
-    targets = bench_random_take);
```

Comment:
looks like we have removed the original bench which runs using the dataset API. can we add that back, so that we have benchmark for taking from dataset vs fragment vs file?

Reply:
I changed the original bench name from bench_random_take to bench_random_take_with_dataset. The implementation is not modified.